gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilExtension.kt

```diff
@@ -17,4 +17,12 @@ open class AnvilExtension {
    * By default this feature is disabled.
    */
   var generateDaggerFactories = false
+
+  /**
+   * There are occasions where consumers of Anvil are only interested in generating Dagger
+   * factories to speed up build times and don't want to make use of the other features. With this
+   * flag Anvil will only generate Dagger factories and skip all other steps. If this flag is set
+   * to `true`, then also [generateDaggerFactories] must be set to `true`.
+   */
+  var generateDaggerFactoriesOnly = false
```

Comment:
I'm not intending to document this flag in the README. Most people probably don't want to use this and I'm only aware of one project using it so far. We can always add it to the documentation when a higher demand is there.